client/js/components/procedure/Shared/SimilarStatementSubmitters/SimilarStatementSubmitters.vue

```diff
@@ -402,6 +425,15 @@ export default {
       }
     },
 
+    showUpdateForm (index) {
+      this.formFields.submitterCity = this.listEntries[index].submitterCity
+      this.formFields.submitterName = this.listEntries[index].submitterName
+      this.formFields.submitterAddress = this.listEntries[index].submitterAddress
+      this.formFields.submitterHouseNumber = this.listEntries[index].submitterHouseNumber
+      this.formFields.submitterPostalCode = this.listEntries[index].submitterPostalCode
+      this.formFields.submitterEmailAddress = this.listEntries[index].submitterEmailAddress
```

Comment:
nitpick: I find this a little less verbose

```suggestion
      const {
        submitterAddress,
        submitterCity,
        submitterEmailAddress,
        submitterHouseNumber,
        submitterName,
        submitterPostalCode
      } = this.listEntries[index]
      
      this.formFields = {
        ...this.formFields, // not sure if this is needed
        submitterAddress,
        submitterCity,
        submitterEmailAddress,
        submitterHouseNumber,
        submitterName,
        submitterPostalCode
      }
```

Reply:
I found a even shorter form ;-) 
e13ad3e588c43000eb5bbcbd90664bc9919e7c04